src/projects/providers/multiplex/multiplex_profile.cpp

```diff
@@ -418,7 +418,7 @@ namespace pvd
 					playlist->SetWebRtcAutoAbr(webrtc_auto_abr.text().as_bool());
 				}
 
-				auto hls_chunklist_path_depth = options_node.child("HlsChunklistPathDepth");
+				auto hls_chunklist_path_depth = options_node.child("HLSChunklistPathDepth") || options_node.child("HlsChunklistPathDepth");
```

Comment:
Here, hls_chunklist_path_depth is a bool type. Is that what you intended?

Reply:
No. It is my fault. I wll fix it